samples/Owin.Nowin.HelloWorld/web.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0"?>
+<configuration>
```

Comment:
IIUC you don't need this file in the project anymore unless you need to customize it, which it doesn't appear we are doing here. Microsoft.NET.Sdk.Web will generate this file.